llvm/test/CodeGen/AArch64/GlobalISel/arm64-atomic.ll

```diff
@@ -2421,7 +2421,7 @@ define i8 @atomicrmw_min_i8(ptr %ptr, i8 %rhs) {
 ; CHECK-NOLSE-O1-NEXT:    ldaxrb w8, [x0]
 ; CHECK-NOLSE-O1-NEXT:    sxtb w9, w8
 ; CHECK-NOLSE-O1-NEXT:    cmp w9, w1, sxtb
-; CHECK-NOLSE-O1-NEXT:    csel w9, w8, w1, le
+; CHECK-NOLSE-O1-NEXT:    csel w9, w8, w1, lt
```

Comment:
lt: Signed less than
le: Signed less than or equal
It is equivalent to ICMP_SLT and ICMP_SLE and thus G_SMIN. The semantics did not change.